docs/UserGuide.md

```diff
@@ -17,13 +17,16 @@ It's optimized for use via a Command Line Interface (CLI) while still having the
 
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` or above installed in your Computer. [How to check](https://www.baeldung.com/java-check-is-installed)
+    - If you do not have Java `11`, download it from [here](https://www.oracle.com/java/technologies/javase/jdk11-archive-downloads.html)
+    - Help with configuring correct Java version: [Windows](https://www.happycoders.eu/java/how-to-switch-multiple-java-versions-windows/) | [MacOS](https://stackoverflow.com/questions/21964709/how-to-set-or-change-the-default-java-jdk-version-on-macos) | [Linux](https://www.baeldung.com/linux/java-choose-default-version)
+    - For Mac users, you are recommended to follow this [guide](https://nus-cs2103-ay2324s2.github.io/website/admin/programmingLanguages.html#programming-language) if you face issues running the `tether.jar` file, notably unreadable or garbled text in the application.
```

Comment:
Perhaps this can come first? So mac users download the right Java 11 from the start